docs/dev-system-overview.md

```diff
@@ -3,18 +3,33 @@ id: system-overview
 title: System Overview
 ---
 
-(From Discord)
+# [The Client](https://github.com/mozilla/hubs) 
+Renders almost everything the user sees. The html pages are served by retculum but come from the client.
```

Comment:
The front-end is built with React for 2D components and Three.js and A-Frame for the 3D scene. The 3D content is synced across clients using [networked aframe](https://github.com/networked-aframe/networked-aframe). Physics run on the client using Ammo.js/Wasm.

Each page on the client is served from the back-end, Reticulum. The `/admin` folder contains the code for the Admin Panel which is separate to run from Hubs.